openlibrary/i18n/fr/messages.po

```diff
@@ -465,78 +473,299 @@ msgstr ""
+#: edit/edition.html:448 view.html:361
 msgid "ID Numbers"
 msgstr "Identifiants"
 
+#: view.html:385
+#, python-format
+msgid "This book was generously donated by %(donor)s"
+msgstr "Ce livre a été généreusement donné par %(donor)s"
+
+#: view.html:387
+msgid "This book was generously donated anonymously"
+msgstr "Ce livre a été généreusement donné de manière anonyme"
+
+#: view.html:392
+msgid "Learn more about Book Sponsorship"
+msgstr "En savoir plus sur le parrainage de livres"
+
+#: edit.html:14
+msgid "Edit List"
+msgstr "Modifier la liste"
+
+#: edit.html:25
+msgid "Label"
+msgstr "Étiquette"
+
+#: edit.html:34 edit.html:54
+msgid "Description"
+msgstr "Description"
+
+#: embed.html:17 home.html:48 preview.html:23 snippet.html:18 view.html:32
+#, python-format
+msgid "1 seed"
+msgid_plural "%(count)d seeds"
+msgstr[0] "1 article"
+msgstr[1] "%(count)d articles"
+
+#: embed.html:22
+msgid "Visit Open Library"
+msgstr "Visiter Open Library"
+
+#: embed.html:22
+msgid "Open Library logo"
+msgstr "Logo Open Library"
+
+#: books.html:86 embed.html:27 home.html:7 home.html:10 lists.html:10
+#: nav_head.html:48 nav_head.html:73 search_head.html:70 view.html:31
+#: view.html:42 view.html:60 widget.html:234
+msgid "Lists"
+msgstr "Listes"
+
+#: embed.html:41 view.html:165
+msgid "Unknown authors"
+msgstr "Auteurs inconnu"
+
+#: embed.html:67
+msgid ""
+"Open in online Book Reader. Downloads available in ePub, DAISY, PDF, TXT "
+"formats from main book page"
+msgstr ""
+"Ouvrir dans le lecteur de livres en ligne. Téléchargements disponibles "
+"aux formats ePub, DAISY, PDF, TXT à partir de la page principale du livre"
+
+#: custom_carousel.html:42 custom_carousel.html:47 editions_datatable.html:23
+#: embed.html:69 openlibrary/macros/ReadButton.html:13 show.html:34
+msgid "Read"
+msgstr "Lire"
+
+#: embed.html:73
+msgid "This book is checked out"
+msgstr "Ce livre est prêté."
+
+#: embed.html:75
+msgid "Checked out"
+msgstr "Prêté"
+
+#: embed.html:78
+msgid "Borrow book"
+msgstr "Emprunter livre"
+
+#: custom_carousel.html:39 custom_carousel.html:45 embed.html:80
+#: openlibrary/macros/ReadButton.html:9
+msgid "Borrow"
+msgstr "Emprunter"
+
+#: embed.html:83 view.html:228 view.html:242
+msgid "Protected DAISY"
+msgstr "DAISY protégé"
+
+#: embed.html:85
+msgid "DAISY"
+msgstr "DAISY"
+
+#: embed.html:100 view.html:215
+#, python-format
+msgid "by %(name)s"
+msgstr "par %(name)s"
+
+#: embed.html:113 view.html:222
+#, python-format
+msgid "- 1 edition"
+msgid_plural "- %(count)s editions"
+msgstr[0] "1 édition"
+msgstr[1] "%(count)s éditions"
+
+#: embed.html:115 view.html:224
+#, python-format
+msgid "- 1 work"
+msgid_plural "- %(count)s works"
+msgstr[0] "1 œuvre"
+msgstr[1] "%(count)s œuvres"
+
+#: exports.html:30
+msgid "Export"
+msgstr "Exporter"
+
+#: exports.html:31
+#, python-format
+msgid "as %(json_link)s, %(html_link)s, or %(bibtex_link)s"
+msgstr "au format %(json_link)s, %(html_link)s, ou %(bibtex_link)s"
```

Comment:
TIL! French doesn't use the oxford comma http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?Th=2&t1=&id=3449
```suggestion
msgstr "au format %(json_link)s, %(html_link)s ou %(bibtex_link)s"
```